tests/testProjectedLikelihood.py

```diff
@@ -191,8 +192,7 @@ def testDirect(self):
         ctrl = lsst.meas.modelfit.UnitTransformedLikelihoodControl()
         var = numpy.random.rand(self.bbox0.getHeight(), self.bbox0.getWidth()) + 2.0
         self.exposure0.getMaskedImage().getVariance().getArray()[:, :] = var
-        efv = lsst.meas.modelfit.EpochFootprintVector()
-        efv.push_back(lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0))
+        efv = [lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0)]
```

Comment:
I'm really liking actually having Python list and tuple types. It looks and acts so much better. Although I'm sure there is a performance penalty somewhere which we have yet to identify and workaround with `ndarray`.